mobify/progressive-web-scaffold
# Astro: merlins checkout sign in buttons

On check out cart implemented hooks so that, on continue button clicked or sign-in clicked, the native app could respond accordingly

 **JIRA**: https://mobify.atlassian.net/browse/HYB-985

## Changes
- cart product list : sign in button action added
- actions.js : added in two actions, onContinue and onSignIn
- container.js : added in two actions to 3 buttons. 2 onContinue and 1 signIn

## How to test-drive this PR
- open up check out modal by clicking the cart button on the header
- clicking continue shopping or sign-in should hide the modal
